CHANGELOG.rst

```diff
@@ -1,6 +1,55 @@
 Changelog
 =========
 
+3.8.0
+-----
+Features:
```

Comment:
How about mentioning that this are the changes since 3.7.3, and not since 3.8.0-rc.1? Something like
```suggestion

This describes the changes since 3.7.3.

Features:
```

Or maybe even remove the 3.8.0-rc.1 entry completely? After all this was only a pre-release, and everyone interested in what exactly was new in there can still check out https://github.com/getsops/sops/blob/v3.8.0-rc.1/CHANGELOG.rst.

Reply:
Updated, thanks for the suggestion :bow: